extensions/sql-database-projects/src/common/telemetry.ts

```diff
@@ -35,5 +36,8 @@ export enum TelemetryActions {
 	changePlatformType = 'changePlatformType',
 	updateSystemDatabaseReferencesInProjFile = 'updateSystemDatabaseReferencesInProjFile',
 	startAddSqlBinding = 'startAddSqlBinding',
-	finishAddSqlBinding = 'finishAddSqlBinding'
+	finishAddSqlBinding = 'finishAddSqlBinding',
```

Comment:
Probably not in scope for this PR but for the start/finish telemetry points - are these to check if the operation was successful or for calculating perf?